@here/harp-datasource-protocol/lib/TileInfo.ts

```diff
@@ -249,6 +293,36 @@ export class ExtendedTileInfo implements TileInfo {
             this.typeCatalog = new Array<string>();
         }
     }
+
+    /**
+     * Compute the memory footprint caused by objects owned by the `ExtendedTileInfo`.
+     */
+    getNumBytes(): number {
+        let numBytes = 100;
+        for (const str of this.textCatalog) {
+            numBytes += 2 * str.length;
+        }
+
+        numBytes += this.techniqueCatalog.length * 100;
```

Comment:
And this 100 here? If this numbers come from somewhere, please define them as constants as the beginning of the file for better readability.

Reply:
Added constants